ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -61,14 +61,20 @@ import SwiftUI
 /// View for List Cells
 struct MSFListCellView: View {
```

Comment:
Why isn't the CellView and the PersonaView public? Consumers of FluentUI won't be able to use in the SwiftUI environment.

Reply:
Resolved offline: windowProvider is not a public protocol, so CellView could not be made public.

Reply:
Yes. I'm preparing a change to pass the windowProvider in a way that doesn't require the init parameter.
We'll make them public as part of that change.